spack/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg_attr(test, feature(test))]
+#![feature(or_patterns)]
```

Comment:
Is this really required?

Reply:
swc uses nightly for a while because of some rustfmt options, but it does not rely on nightly features from non-test codes.

Reply:
not required, I just noticed that nightly was used anyway so I wanted to use some terser syntax, i'm not familiar with the consequences of using nightly features. looking at rust-lang/rust/issues/54883 looks to be out in next stable

should I remove it?

Reply:
It's okay if it's going to be stabilized. Thanks!